### DISCUSSION TITLE
How to change the gray background onHover

### DISCUSSION THREAD

**Discussion by allan-vera** [recharts/recharts#5265] [Q&A] [answered]
Hello. When you hover over a chart this gray background gets put in by default. I haven't seen or figured out a way to recolor it. Is this possible? 

Thanks! 

![Screenshot 2024-11-20 at 5 14 44 PM](https://github.com/user-attachments/assets/df5fc8e5-1a10-4450-9315-bbb98ee4c5fc)
 the chart

**Comment by ckifer** [comment, marked as answer]
Hi try using the cursor prop on Tooltip. 

cursor={{ fill: "blue" }}

**Reply by ckifer** [reply]
Additionally you can do something more custom like this https://github.com/recharts/recharts/issues/5243#issuecomment-2478060185

**Reply by allan-vera** [reply]
WOOO thank you! That's exactly what i was looking for!!

**Reply by ckifer** [reply]
no problem, glad to help!
